reconcile/utils/ocm.py

```diff
@@ -553,7 +588,7 @@ def _ready_for_app_interface(cluster: dict[str, Any]) -> bool:
 
     def _init_clusters(self, init_provision_shards):
         api = f"{CS_API_BASE}/v1/clusters"
-        clusters = self._get_json(api)["items"]
+        clusters = self._get_json(api).get("items")
```

Comment:
Just curious. Can `items` be missing?

Reply:
I think this was a bug since the `_get_json` method supports pagination. `items` is not missing in the API response but the `_get_json` method does not add it if it's empty in the response. 

I think It would be better to return an empty `items` list the same way the API does, but using `get("items")` is use in other methods of the class.